docs/learn.md

```diff
@@ -205,13 +205,13 @@ While Steampipe plugins provide an easy way to query your configuration, Steampi
 
 Steampipe **dashboards** allow you to visualize your steampipe data.
 
-Lets download the AWS Insights mod and view the dashboards.  First, lets clone the repo:
+Let's download the AWS Insights mod and view the dashboards.  First, let's clone the repo:
 
 ```bash
 git clone https://github.com/turbot/steampipe-mod-aws-insights.git
 ```
 
-Now lets change to that directory and run `steampipe dashboard`:
+Now, let's change to that directory and run the `steampipe dashboard`:
```

Comment:
remove `the` and the comma  - should be: Now let's change to that directory and run `steampipe dashboard`: